### DISCUSSION TITLE
Any way to work out what kitten icat thinks it is being passed?

### DISCUSSION THREAD

**Discussion by hectorBrown** [kovidgoyal/kitty#6081] [Q&A] [answered]
I use icat to do image previews in my file manager [lf](https://githhttps://github.com/gokcehan/lfub.com/gokcehan/lf) which worked great until very recently (made a change of distro).

Specifically the command I use is `kitten icat --transfer-mode file --place "${w}x${h}@${x}x${y}" "$file"`. When reading stderr I get the output `Error: The --place option can only be used with a single image, not 2` inside lf. I have tried echoing all the arguments of the command, and running them directly in my terminal works fine.

I think it might help me to work out what string icat is being passed as a second filepath if anyone knows how to do that? Or if anyone has dealt with something similar before.

Thanks!

**Comment by page-down** [comment, marked as answer]
dup of https://github.com/kovidgoyal/kitty/issues/6010

Your preview script is outdated.
https://github.com/gokcehan/lf/issues/1099
https://github.com/gokcehan/lf/wiki/Previews#with-kitty-and-pistol

**Reply by hectorBrown** [reply]
Ah yeah that works perfectly thanks. Probably should have checked issues here, just hadn't imagined it would have been related to a kitty update.
